### DISCUSSION TITLE
Appending text to a users message with bot.

### DISCUSSION THREAD

**Discussion by ggrantrowberry** [discordjs/discord.js#8683] [Q&A] [answered]
I want to create a slash command that appends text to a user's message. This would be similar to the /tableflip, /shrug or even the /giphy built in commands. Is there any way to do this? I have tried with the interaction.reply but it creates a new message that comes from the bot. I need the user to be able to edit the message afterwards.

**Comment by DTrombett** [comment, marked as answer]
This is not supported by Discord at the moment so you should create an issue in the Discord API repo.
At the moment the closest solution you could take is to reply with a webhook using the user's avatar and name. This however shows the (unverified) bot badge near the name and the role color does not appear

**Reply by ggrantrowberry** [reply]
Shoot. Thanks for the reply. Do you have a link to that repo by any chance? I'm not sure which one it is.

**Reply by Jiralite** [reply]
https://github.com/discord/discord-api-docs
